Sources/CreateAPI/Generator/Generator+DataTypes.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import OpenAPIKit30
+
+extension Generator {
+    
+    func stringType(for format: JSONTypeFormat.StringFormat) -> TypeIdentifier {
+        switch format {
+        case .byte:
+            return builtInType("Data", format: "byte", overrides: options.dataTypes.string)
+        case .date:
+            if options.useNaiveDate {
+                setNaiveDateNeeded()
+                return .builtin("NaiveDate")
+            }
+            return builtInType("String", format: "date", overrides: options.dataTypes.string)
+        case .dateTime:
+            return builtInType("Date", format: "date-time", overrides: options.dataTypes.string)
+        case .other(let format) where format == "uri":
+            return builtInType("URL", format: "uri", overrides: options.dataTypes.string)
+        case .other(let other) where other == "uuid":
+            return builtInType("UUID", format: "uuid", overrides: options.dataTypes.string)
```

Comment:
Switch actually lets you write that where clause inline if you want

```suggestion
        case .other("uri"):
            return builtInType("URL", format: "uri", overrides: options.dataTypes.string)
        case .other("uuid"):
            return builtInType("UUID", format: "uuid", overrides: options.dataTypes.string)
```